Documentation/ApiOverview/PageTypes/Index.rst

```diff
@@ -232,3 +245,15 @@ need to add the new doktype as select item and associate it with the configured
       );
    })();
 
+As you can see from the example, to make sure you get the correct icons, you can utilize :php:`typeicon_classes`.
+
+For the following cases you need to configure icons explictly, otherwise they will automatically fall back to the
+variant for regular page doktypes.
+
+* Page is hidden in navigation
+* Page is site-root
+* Page contains content from another page
```

Comment:
Here we should mention the implied icon identifier suffix explicitly once more for each case.